its/projects/IncrementalPRAnalysis/IncrementalPRAnalysis/File1.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace IncrementalPRAnalysis
```

Comment:
We can use file scope namespaces.

Reply:
Not for `QA: Run ITs vs2017_latest89`

Reply:
..and still can target net6.0.. I'll try

Reply:
ok, worked